src/AccessibleGooglePlacesAutocomplete.tsx

```diff
@@ -101,9 +100,31 @@ export class AccessibleGooglePlacesAutocomplete extends React.Component<
     this.getStatusNoResultsMessage = this.getStatusNoResultsMessage.bind(this);
   }
 
+  public onAutoCompleteSelect = (value: string) => {
+    this.predictions.forEach(element => {
+      if (element.description === value) {
```

Comment:
Do you want to run a the PlacesService on each matched prediction or just the first?

To run on just the first match use
```js
const selectedPrediction = array.find(prediction => prediction => description === value);
...
```

To run on all matched predictions use:
```js
predictions
  .filter(prediction => prediction => description === value)
  .map(prediction => { ... });
```